src/ralph/helpers.py

```diff
@@ -9,6 +9,27 @@
 logger = logging.getLogger(__name__)
 
 
+def dnsaas_txt_record_data(hostname, target_owner, owner, data):
+    """Return dict with data set for dnsaas auto txt records update
+
+    target_owner: owner of txt record (as username)
+    owner: dnsaas user responsible for update
+    data = (
+        (key, value),
+        ..
+    )
+    """
+    publish_data = []
+    for purpose, content in data:
+        data = {
+            'name': hostname,
+            'target_owner': target_owner,
+            'owner': owner,
+            'purpose': purpose,
+            'content': content,
+        }
+    return publish_data
```

Comment:
I don't get this function. What's the purpose of it? And `publish_data` will always be `[]`